jstest/builder/utils.py

```diff
@@ -43,9 +58,91 @@ def create_build_info(env):
     # Merge the collected values into a result object.
     build_info = {
         'build-date': utils.current_date('%Y-%m-%dT%H.%M.%SZ'),
+        'build-date': get_standardized_date(),
```

Comment:
You will overwrite the first `build-date` above. But I think this patch was not properly rebased to the master. The `get_standardized_date` function was removed in #177.